google/cloud/bigtable/cluster.py

```diff
@@ -104,10 +133,13 @@ def __init__(
         self.default_storage_type = default_storage_type
         self._kms_key_name = kms_key_name
         self._state = _state
+        self.min_serve_nodes = min_serve_nodes
```

Comment:
do you think it would be a good idea to do some validation here? Here are some scenarios that come to mind:
- The user sets serve nodes AND autoscaling parameters
- The user sets no settings relating to auto or manual scaling
- The user only sets one or two (not all three) autoscaling config parameters

Reply:
Great suggestion. I added the `_validate_scaling_config` function, which gets called in `create` and `update`. I also added unit tests to test the above scenarios.

What do you think about this?